inference-engine/tests/functional/shared_test_classes/src/single_layer/detection_output.cpp

```diff
@@ -138,15 +159,25 @@ void DetectionOutputLayerTest::SetUp() {
     std::tie(attrs.variance_encoded_in_target, attrs.share_location, attrs.normalized, attrs.input_height, attrs.input_width,
              inShapes[idxLocation], inShapes[idxConfidence], inShapes[idxPriors], inShapes[idxArmConfidence], inShapes[idxArmLocation]) = specificAttrs;
 
-    if (inShapes[idxArmConfidence].empty()) {
+    if (inShapes[idxArmConfidence].first.rank().get_length() == 0) {
         inShapes.resize(3);
     }
 
-    for (size_t i = 0; i < inShapes.size(); i++) {
-        inShapes[i][0] = batch;
+    for (auto& value : inShapes) {
+        auto shapes = value.second;
+        for (auto& shape : shapes) {
+            shape[0] = batch;
+        }
     }
 
-    auto params = ngraph::builder::makeParams(ngraph::element::f32, inShapes);
+    init_input_shapes({ inShapes });
+
+    std::vector<ov::PartialShape> inPartialShapes;
+    inPartialShapes.reserve(inShapes.size());
+    for (auto it = inShapes.begin(); it != inShapes.end(); ++it) {
+        inPartialShapes.push_back(it->first);
+    }
+    auto params = ngraph::builder::makeDynamicParams(ngraph::element::f32, inPartialShapes);
```

Comment:
Why can't we just use `inputDynamicShapes` here?

Reply:
there is no input dynamic/static PartialShapes (`std::vector<ov::PartialShape>`) prepared collection in this tests

Reply:
done, thanks